adoc/deployment-sles.adoc

```diff
@@ -0,0 +1,15 @@
+== Deployment on existing SLES installation
+
+If you already have a running {sle} 15 SP1 installation, you can add {productname}
+to this installation using SUSE Connect. You also need to enable the "Server Applications"
+and "Containers" modules because they contains some dependencies required by
+{productname}.
+
+Retrieve your {productname} registration code and run:
+
+----
+SUSEConnect -p sle-module-containers/15.1/x86_64
+SUSEConnect -p sle-module-server-applications/15.1/x86_64
```

Comment:
do we need server-applications?

Reply:
@jordimassaguerpla There was a package that was moved from server applications to base system right? If so let's remove this requirement.

Reply:
exactly, socat. But now server applications is not needed anymore.